app/keepers/keepers.go

```diff
@@ -443,6 +438,11 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 
 	appKeepers.ValidatorSetPreferenceKeeper = &validatorSetPreferenceKeeper
 
+	appKeepers.SuperfluidKeeper = superfluidkeeper.NewKeeper(
```

Comment:
Lol oof, were really going to have to come back to superfluid at some point and clean it up. Poor module becoming so omnibus

Reply:
yeah :( poor superfluid keeper :(